packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -93,6 +95,8 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
   private _isScrollIdle: boolean;
   private readonly _navigationIdleDelay: number = 250 /* ms */;
```

Comment:
We should just use const at the top of the file for constant numbers.

/** Comment explaining what its used for. */
const NavigationIdleDelay = 250;